CMakeLists.txt

```diff
@@ -11,6 +10,8 @@ cmake_minimum_required (VERSION 2.8.11)
 #
 #  THESE PATHS MAY NEED TO BE CHANGED DEPENDING ON WHERE THEY ARE STORED!!!
 #
+option(NALU_USES_OPENFAST
```

Comment:
Need to coordinate with @jrood-nrel on the nightly testing for merging this pull request.